core/src/main/java/org/elasticsearch/action/search/SearchRequest.java

```diff
@@ -58,6 +58,8 @@
 
     private static final ToXContent.Params FORMAT_PARAMS = new ToXContent.MapParams(Collections.singletonMap("pretty", "false"));
 
+    public static final int DEFAULT_PRE_FILTER_SHARDS_AFTER = 1;
```

Comment:
haha yeah true I wanted to trigger this constantly so I changed this but didn't revert